### PR TITLE
Add Tabs documentation page

### PR DESCRIPTION
**Pull request by farazcsk**

### CODE REVIEW

File: styleguide/screens/Patterns/TabBar/TabBar.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import TabBarIconItem from '../../../../components/Navigation/TabBar/TabBarIconI
 
 import m from '../../../../globals/modifiers.css';
 
-const BadgeDocumentation = () => (
+const TabBarDocumentation = () => (
```
Comment (rdjpalmer) [inline styleguide/screens/Patterns/TabBar/TabBar.js]: Oops (I'm awful for this)

File: styleguide/screens/Patterns/TabBar/TabBar.js [hunk 2]
```diff
@@ -66,7 +66,6 @@ const BadgeDocumentation = () => (
           </TabBarItem>
           <TabBarItem
             href="#notification"
-            active
```
Comment (rdjpalmer) [inline styleguide/screens/Patterns/TabBar/TabBar.js]: How come you removed this?

File: styleguide/screens/Patterns/Tabs/Tabs.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+import Tab from '../../../../components/Tabs/Tab';
+
+import m from '../../../../globals/modifiers.css';
+
+const TabsDocumentation = () => (
+  <div>
+    <H level={ 1 }>Loader</H>
+    <T elm="p" className={ cx(m.mtr, m.largeI, m.demi) }>
+      <C>Tabs</C> provide an easy way to navigate between views on a single page,
+      where content is related. For cases when you need page level tabs, <C>TabBar</C>.
```
Comment (rdjpalmer) [inline styleguide/screens/Patterns/Tabs/Tabs.js]: `TabBar` here should be a link to the other documentation... we should add a link going the other way too

File: styleguide/screens/Patterns/Tabs/Tabs.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    </T>
+    <D>
+      <Specimen
+        classNames={ {
+          specimenContainer: m.par,
+        } }
+        code={ dedent`
+          <Tabs>
+            <Tab label="One">
+              One
```
Comment (rdjpalmer) [inline styleguide/screens/Patterns/Tabs/Tabs.js]: Could we make it more clear that the "One" and "Two" get rendered as the body of the tab?

File: styleguide/screens/Patterns/TabBar/TabBar.js (hunk 2 above)
Comment (farazcsk) [reply styleguide/screens/Patterns/TabBar/TabBar.js]: I could be wrong but is that not how they are intended to be used? i.e., only one tab can be active at one time. More than one active tab item throws a warning and breaks the tests.

File: styleguide/screens/Patterns/Tabs/Tabs.js (hunk 3 above)
Comment (farazcsk) [reply styleguide/screens/Patterns/Tabs/Tabs.js]: Will add 👍

File: styleguide/screens/Patterns/Tabs/Tabs.js (hunk 4 above)
Comment (farazcsk) [reply styleguide/screens/Patterns/Tabs/Tabs.js]: I could change the tab content to "body one" and "body two"?

File: styleguide/screens/Patterns/TabBar/TabBar.js (hunk 1 above)
Comment (farazcsk) [reply styleguide/screens/Patterns/TabBar/TabBar.js]: I definitely did not notice it in the PR 😆

File: styleguide/screens/Patterns/TabBar/TabBar.js (hunk 2 above)
Comment (rdjpalmer) [reply styleguide/screens/Patterns/TabBar/TabBar.js]: Turns out it's really hard to tell from the diff that you're not removing the one correct usage of it 😅

File: styleguide/screens/Patterns/Tabs/Tabs.js (hunk 4 above)
Comment (rdjpalmer) [reply styleguide/screens/Patterns/Tabs/Tabs.js]: Sure. Can see how that looks 👍

**Pull request merged**